float64 thrustRoll
float64 thrustPitch
float64 thrustYaw
float64 Throttle